js/Plan.jsx

```diff
@@ -22,7 +20,8 @@ var store = Redux.createStore(
 
 var Plan = function(props) {
 
-	window.api_url = props.model.APIEndpoint;
+	window.api_url = props.model.api_endpoint;
```

Comment:
Yes, I was thinking that this enpoint would probably end up as 
`http://deploy.silverstripe.dev/naut/project/cbmicro/environment/Production/`
and the javascript api module would then add the dispatcher name to the url for the individual calls.

current api_endpoint is locked down to `http://deploy.silverstripe.dev/naut/project/cbmicro/environment/Production/plan`